File: testprojects/python_main/pipelines/PYTHON_DEP_MGMT_ALL/code/com/main1/pythondepmanagement_1/graph/SQLStatement_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from com.main1.pythondepmanagement_1.config.ConfigStore import *
from com.main1.pythondepmanagement_1.udfs.UDFs import *

def SQLStatement_1(spark: SparkSession, in0: DataFrame) -> (DataFrame, DataFrame):
    in0.createOrReplaceTempView("in0")
    df1 = spark.sql("select * from in0 where in0.`c  - int` > '$c_0' and in0.`c- short` > 1")
    df2 = spark.sql("select * from in0 where in0.`c  - int` != '$c_int' and in0.`c-string` not like '$c_sql_pattern'")

    return df1, df2

File: testprojects/master_scala_mainone/pipelines/SCALA_DEPENDENCY_MANAGEMENT/code/src/main/scala/graph/SQLStatement_1.scala
package graph

import io.prophecy.libs._
import udfs.UDFs._
import config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object SQLStatement_1 {

  def apply(context: Context, in0: DataFrame): (DataFrame, DataFrame) = {
    in0.createOrReplaceTempView("in0")
    (context.spark.sql(
       "select * from in0 where in0.customer_id not like '$c_sql_expr'"
     ),
     context.spark.sql(
       "select * from in0 where in0.first_name not like '$c_sql_expr'"
     )
    )
  }

}
